craco.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    {
+      plugin: require("craco-cesium")()
```

Comment:
We need this plugin for cesium support.

Reply:
Should we locate this in the `node_modules` directory where `solr-faceted-search-react` lives?   Why or why not?

Reply:
I found two useful resources about it. https://marmelab.com/blog/2021/07/22/cra-webpack-no-eject.html for why we need craco. And https://github.com/gsoft-inc/craco for the craco documentations.

Reply:
Makes sense -- I didn't realize it was designed to work *with* create-react-app.